### ISSUE TITLE
only ground mounts being chosen in WOTLK zones

### ISSUE THREAD

**Issue by hivehum** [Bestride/BeStride#205]
### WoW Version

Retail (9.2.5)

### Bestride Version

2.0.6

### Steps to Reproduce

type /click BeStride_ABRegularMount in any WOTLK zone (personally verified in dalaran, icecrown, scholazar basin, and borean tundra) with flying mounts selected in the /br menu

### Observed Behavior

only ground mounts will be summoned despite being able to fly in these zones

**Comment by hivehum** [comment]
with the newest version of retail and version v.2.0.7-alpha1 of bestride, /click BeStride_ABRegularMount does not function at all for me

**Comment by DanSheps** [comment]
It does work for me.

Can you open the options?

**Comment by hivehum** [comment]
yes, i can option the options and alter them freely. but i can't use the addon to summon mounts

**Comment by sscree** [comment]
I get this, too with bestride 2.0.6 on retail 10.0.2. The problem is that the IsFlyable() function in logic.zone is looking for Cold Weather Flying (54197) when continent.mapID == 113. However IsSpellKnown(54197) returns false. Cold Weather Flying is no longer required to fly in WotLK zones in retail. If you comment out lines 163-165 in BeStride_Constants, then bestride is able to pick flying mounts in WotLK zones.

**Comment by walter-hwhite** [comment]
Confiming the same behaviour in version 2.09. Only noticed as I was levelling some alts.
